Playbooks/playbook-Email_Address_Enrichment_-_Generic_v2.1.yml

```diff
@@ -0,0 +1,598 @@
+      version: -1
+      name: Is there a domain list input?
+      description: Checks if there is at least one domain to check for domain-squatting.
+      scriptName: Exists
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "4"
+      "yes":
+      - "15"
+    scriptarguments:
+      value:
+        complex:
+          root: inputs.Domain
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 100.5,
+          "y": 960
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "20":
+    id: "20"
+    taskid: 3ecf1917-78f6-4350-84bb-704fdaa1bf05
+    type: title
+    task:
+      id: 3ecf1917-78f6-4350-84bb-704fdaa1bf05
+      version: -1
+      name: No Email Addresses
+      description: ""
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -380,
+          "y": 785
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "13_11_yes": 0.64,
+      "13_4_#default#": 0.16,
+      "14_10_yes": 0.65,
+      "14_4_#default#": 0.17,
+      "17_4_#default#": 0.23,
+      "18_4_#default#": 0.23,
+      "5_20_#default#": 0.35
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1405,
+        "width": 1432.5,
+        "x": -380,
+        "y": 50
+      }
+    }
+  }
+inputs:
+- key: InternalDomains
+  value:
+    complex:
+      root: inputs.InternalDomains
+      transformers:
+      - operator: uniq
+  required: false
+  description: A CSV list of internal domains. The list will be used to determine
+    whether an email address is internal or external.
+- key: Email
+  value:
+    complex:
+      root: Account
+      accessor: Email.Address
+      transformers:
+      - operator: uniq
+  required: false
+  description: The email addresses to enrich.
+- key: Domain
+  value:
+    complex:
+      root: inputs.Domain
+      transformers:
+      - operator: uniq
+  required: false
+  description: The domains associated with the incident. These domains will be checked
+    for domain-squatting.
+outputs:
+- contextPath: Account
+  description: The Account object.
+  type: unknown
+- contextPath: Account.Email.NetworkType
+  description: The email account NetworkType (Internal/External).
+  type: string
+- contextPath: Account.Email.Distance.Domain
+  description: The compared domain.
+  type: string
+- contextPath: Account.Email.Distance.Value
+  description: 'The distance between the email domain and the compared domain. '
+  type: number
+- contextPath: DBotScore
+  description: The DBotScore object.
+  type: unknown
+releaseNotes: "Fixed a bug where DomainSquatting and Active Directory filters were performing an incorrect check.
```

Comment:
The DomainSquatting and Active Directory filters now work as expected.

Reply:
So only the first line is replaced with the new one you wrote